### ISSUE TITLE
Make POLL_INTERVAL optional

### ISSUE THREAD

**Issue by denar90** [philipwalton/easy-sauce#7]
Problem is related to ngrok. It allows only 20 connections per 1 minute. If I have e.g. 10 platforms then tests on some of them will be failed because of to many connections which are rejected by ngrok.
Thus I propose to make `POLL_INTERVAL` one of the settings param. So I know that I have a lot of platforms and poll POLL_INTERVAL should be increased.
Another idea - make it smart. If platforms number is big the Increase it automatically. But I'm not sure about all mechanism and stuff...

**Comment by philipwalton** [comment]
I recently encountered this problem as well after ngrok changed its quota limits. I reached out to the maintainer of ngrok to see if the limits could be increased, but it doesn't seem like that is going to happen.

In this case, changing `POLL_INTERVAL` will actually not solve the problem because that's the interval at which requests are made to the Sauce Labs API to check the job status (not to localhost through ngrok).

I haven't fully decided how best to solve this problem, but at this point I'm leaning toward just using Sauce Connect.

Hopefully I can make that change in a way that is completely transparent to users of this library.
